### ISSUE TITLE
Null alt attribute

### ISSUE THREAD

**Issue by jim-rosales** [betterplanning111/dasmoto2#3]
Just as a reminder the reason we include the [alt attribute](https://en.wikipedia.org/wiki/Alt_attribute) along with a description is that it is used as a backup in case the image cannot be rendered, and [screen readers](https://www.codecademy.com/articles/accessibility) use it. Visually impaired users will have the image described to them by the screen reader using the description in the attribute. 

Here you used the null attribute by leaving the description `""` so the screen reader will assume the image is not essential content and not describe it to the user. 

If this was your intention your use of the null attribute was correct otherwise we try and always include a description of most images used. 

https://github.com/betterplanning111/dasmoto2/blob/e8b034e49cb990c005650c84681deba5cb0d5977/index.html#L16
